app/models/memberships/join_zusatzsektion.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+module Memberships
+  class JoinZusatzsektion < MemberJoinSectionBase
+
+    def initialize(join_section, person, join_date, family_membership: false, **params)
+      super(join_section, person, join_date, **params)
+      @family_membership = family_membership
+      @group = join_section.group_for_neuanmeldung
+      @created_at = Time.zone.now
+
+      raise 'missing subgroup' unless group
+    end
+
+    private
+
+    def affected_people
+      family_membership? ? super : [person]
+    end
+
+    def prepare_roles(person)
+      group.roles.build(
+        person: person,
+        type: group.class.const_get('NeuanmeldungZusatzsektion'),
+        beitragskategorie: derive_beitragskategorie,
+        created_at: created_at,
+        delete_on: nil
+      )
+    end
+
+    def derive_beitragskategorie
+      if family_membership?
+        :family
+      else
+        SacCas::Beitragskategorie::Calculator.new(person).calculate(consider_family: false)
+      end
+    end
+
+    def family_membership?
```

Comment:
```suggestion
    def sac_family_membership?
```

ich verwende noch gerne den sac prefix ... family ist z.B. auch ein Konstrukt das wir im Core oder Youth wagon kennen (geschwister)